### ISSUE TITLE
mp4decrypt can decrypt aac sample-aes?

### ISSUE THREAD

**Issue by xavierlaffargue** [axiomatic-systems/Bento4#962]
I would like to know if mp4decrypt works with aac audio segment and SAMPLE-AES encryption. Thanks!

**Comment by barbibulle** [comment]
Yes, that's supported. Did you run into an instance where it isn't working as expected?
